manifests/base/argo/workflow-controller-configmap.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
We have `configMapGenerator` and `secretGenerator` which can the pods auto pick up the latest configmaps or secrets if they have changed. You can read more info [here](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/configGeneration.md).

Reply:
Thanks for the doc. It looks great. Based on the doc it seems gen is optional. 
The idea here is to reduce the templating "magic" to minimum and having the spec as close to the plain yaml as possible. 
(it was a hard lesson learnt by using ksonnet and forgive me being overly cautious on it :P)